lib/cred_stash.rb

```diff
@@ -16,9 +16,9 @@ def get(name, context: {}, raise_if_missing: false)
       nil
     end
 
-    def put(name, value, context: {})
+    def put(name, value, kms_key_id: nil, context: {})
```

Comment:
Passing `kms_key_id: nil` is safe because `CredStash::CipherKey` knows what to do with a nil.

(I did refactor `cipher_key.rb` slightly to make it clearer.)